### PR TITLE
 PWGGA/GammaConv/AliAnalysisTaskSigmaPlToProtonPiZero

### PR DESCRIPTION
**Pull request by AMechler**
Changes to SigmaPlToProtonPiZero task

### CODE REVIEW

**Comment by nschmidtALICE** [comment]
+test

**Comment by AMechler** [comment]
Sorry i have two open PRs and now everything is messed up
those arent the right versions i believe

**Comment by AMechler** [comment]
+test

**Comment by AMechler** [comment]
Should be working now

**Comment by dmuhlhei** [comment]
+test

File: PWGGA/GammaConv/AliAnalysisTaskSigmaPlToProtonPiZero.cxx [hunk 1]
```diff
@@ -513,165 +513,152 @@ void AliAnalysisTaskSigmaPlToProtonPiZero::UserExec(Option_t *)
...
+				if((((AliCaloPhotonCuts*)fClusterCutArray->At(iCut))->ClusterIsSelected(clus,fESD,fMCEvent,fIsMC, 1.,i)) == kTRUE){
+					photon.push_back(clus);
+					if(fHistClusterE[iCut]) fHistClusterE[iCut]-> Fill(clus->E());
+					nClusWCuts+= 1;
 				}
-				for(unsigned int iPhoton1 = 0; iPhoton1 < photon.size(); ++iPhoton1) {
-					trueSigmaPhoton1 = kFALSE;
-					if(!photon[iPhoton1]) {
-						Printf("ERROR: Could not find photon[%i][%i]",iCut,iPhoton1);
+				clus = NULL;
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliAnalysisTaskSigmaPlToProtonPiZero.cxx]: What happens to the ESDCluster which is not used for the vector? Shouldn't it be deleted here?

**Comment by dmuhlhei** [comment]
+test

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**